core/css.js

```diff
@@ -142,12 +142,13 @@ Blockly.Css.CONTENT = [
   '  background-color: #fff;',
   '  outline: none;',
   '  overflow: hidden;',  /* IE overflows by default. */
+  '  display: block;', 
   '}',
 
   '.blocklyWidgetDiv {',
   '  display: none;',
   '  position: absolute;',
-  '  z-index: 999;',
+  '  z-index: 99999 !important;',
```

Comment:
The aim is to make it absolutely at front end.
This should not be overwritable.

I'm not an expert with CSS; it is a common workaround for this.

A better solution, would be to move the widget div next to SVG and toolbox.
The problem is that the Widget DIV is unique (single instance for any
worlspace) so moving it is not a good solution.
The really good solition would be to create one widget div for one
workspace instance.

As it is a lot of work, I didn't try it in my first pull-request.






2015-09-22 13:19 GMT+02:00 Carlos <notifications@github.com>:

> In core/css.js
> <https://github.com/google/blockly/pull/181#discussion_r40074777>:
>
> >    '}',
> >
> >    '.blocklyWidgetDiv {',
> >    '  display: none;',
> >    '  position: absolute;',
> > -  '  z-index: 999;',
> > +  '  z-index: 99999 !important;',
>
> Is the !important really necessary? It makes overwriting the css property
> in your own css file not really possible.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/blockly/pull/181/files#r40074777>.
>